examples/sglang_bench/bench_sharegpt.py

```
@@ -384,36 +522,85 @@ def main():
+    config_tp: Optional[int] = None
+    for i, arg in enumerate(server_args):
+        if arg in ("--tp-size", "--tp") and i + 1 < len(server_args):
+            try:
+                config_tp = int(server_args[i + 1])
+            except ValueError:
+                pass
+            break
+
+    effective_tp = config_tp if config_tp is not None else local_gpus
+    if config_tp is None:
+        server_args = server_args + ["--tp-size", str(effective_tp)]
```

Ruff (0.15.9)

[warning] Consider `[*server_args, "--tp-size", str(effective_tp)]` instead of concatenation

Replace with `[*server_args, "--tp-size", str(effective_tp)]`

(RUF005)